.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: python
+python:
+  - "2.7"
+install:
+    - pip install yapf==0.6.2
+script:
+    - yapf --diff --exclude=./server/mlabns/third_party/* --recursive --style=google ./server/mlabns/*
```

Comment:
Suggest just `dev/` or `hooks/` at the root directory. The only goal in placement I think is making it obvious and easy to developers who work on mlab-ns to use the hooks.